src/app/connections/list-toolbar/list-toolbar.component.ts

```diff
@@ -60,7 +62,21 @@ export class ConnectionsListToolbarComponent<T> implements OnInit, OnDestroy {
     } as ToolbarConfig;
 
     this.subscription = this.items
-      .do(connections => this._allItems = connections)
+      .do(items => this._allItems = items)
+      .do(items => {
+        if (_.some(items, item => _.isArray(item['tags']))) {
```

Comment:
You can use ES6 stuff, i.e. `items.filter()`, `items.find()` etc.  I think actually we could get rid of the lodash dependency, I think there's just one bit of code that's using it currently.